### ISSUE TITLE
document about curl POST command is not correct

### ISSUE THREAD

**Issue by nibiwodong** [DependencyTrack/dependency-track#1023]
The defect may already be reported! Please search for the defect before creating one.

### Current Behavior:
![image](https://user-images.githubusercontent.com/17465789/116395765-1f8eb780-a857-11eb-87ca-cb61b31a4ffa.png)
```
curl -X "POST" "http://dtrack.example.com/api/v1/bom" \
     -H 'Content-Type: multipart/form-data; charset=utf-8; boundary=__X_CURL_BOUNDARY__' \
     -H 'X-Api-Key: LPojpCDSsEd4V9Zi6qCWr4KsiF3Konze' \
     -F "project=f90934f5-cb88-47ce-81cb-db06fc67d4b4" \
     -F "bom=<?xml version=\"1.0\" encoding=\"UTF-8\"?>..."
```
http response is 400

when I change Content-Type header, it worked
```
curl -X "POST" "http://dtrack.example.com/api/v1/bom" \
     -H 'Content-Type: multipart/form-data' \
     -H 'X-Api-Key: LPojpCDSsEd4V9Zi6qCWr4KsiF3Konze' \
     -F "project=f90934f5-cb88-47ce-81cb-db06fc67d4b4" \
     -F "bom=<?xml version=\"1.0\" encoding=\"UTF-8\"?>..."
```

### Steps to Reproduce:
modify document, correct the curl command

### Expected Behavior:
```
curl -X "POST" "http://dtrack.example.com/api/v1/bom" \
     -H 'Content-Type: multipart/form-data' \
     -H 'X-Api-Key: LPojpCDSsEd4V9Zi6qCWr4KsiF3Konze' \
     -F "project=f90934f5-cb88-47ce-81cb-db06fc67d4b4" \
     -F "bom=<?xml version=\"1.0\" encoding=\"UTF-8\"?>..."
```
also give more example and let users know the restapi demo, like this
curl -X "POST" "http://dtrack.example.com/api/v1/bom" \
     -H 'Content-Type: multipart/form-data' \
     -H "X-Api-Key: xxxxxxx" \
     -F "autoCreate=true" \
     -F "projectName=xxxx" \
     -F "projectVersion=xxxx" \
     -F "bom=@target/bom.xml"

### Environment:
  
  - Dependency-Track Version:4.2.0
  - Distribution: [ Docker]
  - BOM Format & Version:
  - Database Server: [ H2 | MSSQL | MySQL | PostgreSQL ]
  - Browser: 

### Additional Details:

(e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by kingc1** [comment]
@nibiwodong Thank you for pointing this out.  I ran into the same issue, and this was the fix I was looking for.  +1 for updating the documentation.

**Comment by stevespringett** [comment]
Docs have been updated and will be published with the release of 4.3
